builder/dockerfile/internals_linux.go

```diff
@@ -49,6 +49,15 @@ func parseChownFlag(chown, ctrRootPath string, idMappings *idtools.IDMappings) (
 	return chownPair, nil
 }
 
+func parseChmodFlag(chmodStr string) (uint16, error) {
```

Comment:
needs to parse symbolic names, should mimic most `chmod` flags like `a+rX,ug+w` if not all of chmod.

Reply:
Done